metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -305,11 +335,6 @@ private static EmptyTag getMetaTag(Map<String, String> metadata, String content)
         return meta().withName(content).withContent(metadata.get(content));
     }
 
-    private URI prepareWebServerAndReturnUriToMetadata(String filename) {
```

Comment:
These methods were not in use. Don't know what your policy is on cleaning up this kind of code.